### PR TITLE
Add a toValue() method to member elements

### PR DESCRIPTION
**Pull request by danielgtaylor**
This is useful for elements that aren't objects but that use members, for example HTTP headers in the API Description namespace. The element is an array of member elements, and when converted to a js value it would be nice to get out an array of tuples.

cc @smizell

### CODE REVIEW

File: lib/primitives/member-element.js [hunk 1]
```diff
@@ -12,6 +12,10 @@ module.exports = function(BaseElement, registry) {
       this.element = 'member';
     },
 
+    toValue: function () {
+      return [this.key.toValue(), this.value.toValue()];
+    },
```
Comment (smizell) [inline lib/primitives/member-element.js:17]: Question: do you want to return this array or would you rather have a hash?

``` js
return {
  key: this.key.toValue(),
  value: this. value.toValue()
}
```

**Comment by smizell** [comment]
@danielgtaylor My only question is tuple vs. hash. Any specific reason to go this route?

**Comment by smizell** [comment]
@pksunkara I think Daniel is away. Interested in commenting on my question above?

**Comment by pksunkara** [comment]
I think hash is better.

**Comment by smizell** [comment]
@pksunkara I think I agree. Reason being, if I get a tuple, I'm going to convert it to variables anyway to make it semantically understandable (i.e. I don't want `member[0]` all over my code when it could say `key`).

**Comment by pksunkara** [comment]
Yup, my reasoning was the same.

**Comment by smizell** [comment]
@pksunkara Would you like for me to change this, or would you like to wait for Daniel to return?

**Comment by pksunkara** [comment]
I don't mind it anyway. If you are busy, I can do it later.

**Comment by smizell** [comment]
@pksunkara I cannot get to it for a while today. If you end up handling it, please ping me so I don't start :) I'll do the same.

**Comment by pksunkara** [comment]
I meant, I can do it next week when I finish up my current task for updating protagonist.

**Comment by pksunkara** [comment]
PR opened at #76

**Pull request closed without merging**